capstone/src/main/webapp/youtubeScript.js

```diff
@@ -162,15 +202,83 @@ function showAvailableAttributes() {
     label.appendChild(document.createTextNode(attribute));
     avaiableAttributesElement.appendChild(checkbox);
     avaiableAttributesElement.appendChild(label);
-    avaiableAttributesElement.appendChild (document.createTextNode (" "));
+    avaiableAttributesElement.appendChild(document.createTextNode (" "));
   });
 }
 
 /** Checks if a character is a letter */
 function isLetter(character) {
-  if ((character.charCodeAt() >= 65 && character.charCodeAt() <= 90) || (character.charCodeAt() >= 97 && character.charCodeAt() <= 122)) {
-    return true;
-  } else {
-    return false;
+  return (character.charCodeAt() >= 65 && character.charCodeAt() <= 90) || (character.charCodeAt() >= 97 && character.charCodeAt() <= 122); 
+}
+
```

Comment:
Let's add some comments explaining what this function and the ones below it do

Reply:
addressed in "address chris and daniel nits"